.release-notes/4173.md

```diff
@@ -0,0 +1,18 @@
+## Fix for segfault when using Array.copy_to() with an empty Array
+
+Currently, the `Array.copy_to()` function does not check whether the Array is empty or has been instanced with any values. This leads to a runtime crash when calling `copy_to()` on an empty array such as in this example (issue #4172):
```

Comment:
You can't say currently here. Release notes are after the change it made. This would be...